### PR TITLE
2021.r1.ssc268 from patch into develop

### PR DESCRIPTION
**Pull request by sjanzou**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -65,13 +65,13 @@ jobs:
       shell: bash {0}
       run: |
         git ls-remote --heads --exit-code https://github.com/NREL/lk.git $GIT_BRANCH
-        if [[ $? != "0" ]]; then echo "LK_BRANCH=develop" >> $GITHUB_ENV; else echo "LK_BRANCH=$GIT_BRANCH" >> $GITHUB_ENV; fi
+        if [[ $? != "0" ]]; then echo "LK_BRANCH=patch" >> $GITHUB_ENV; else echo "LK_BRANCH=$GIT_BRANCH" >> $GITHUB_ENV; fi
```
Comment (brtietz) [inline .github/workflows/ci.yml:68]: We had a bit of a mess where someone branched from develop into patch, which required another commit to set these values to patch, and now we need yet another commit to set these to develop again. I think the correct steps are:

(1) Create a branch off of develop
(2) Merge patch into that branch
(3) Correct these branch names in a new commit
(4) Merge that branch into develop

I'll close this PR, do 1-3, and then send it to you for approval.

**Review by brtietz — CHANGES_REQUESTED** [review]
Need to do a more complicated merge procedure due to CI files.

**Comment by brtietz** [comment]
Closing in favor of https://github.com/NREL/SAM/pull/956

File: .github/workflows/ci.yml (hunk 1 above)
Comment (sjanzou) [reply .github/workflows/ci.yml:68]: Thanks for this - maybe docHub?

**Pull request closed without merging**